src/gui_common/tooltip/DefaultToolTip.cs

```diff
@@ -62,7 +62,10 @@ public bool ToolTipVisible
 
     public override void _Ready()
     {
-        descriptionLabel = GetNode<Label>(DescriptionLabelPath);
+        // For some reason the NodePath wouldn't set correctly if the scene is instantiated with
+        // a different node name, so this use hard-coded path for now
+        descriptionLabel = GetNode<Label>("MarginContainer/VBoxContainer/Description");
```

Comment:
If this is still the case, add a TODO comment here, and open an issue on Github about this. This might catch someone working on the GUI off guard otherwise.

Reply:
#1855